README.md

```diff
@@ -155,11 +155,12 @@ The `th2_check1_active_tasks_number` metric separate rules with label `rule_type
 + `auto-silence-check-after-sequence-rule` to setup a default behavior for CheckSequenceRule
 + New configuration parameter `time-precision` witch is uses if the user has not specified a time precision
 + New configuration parameter `decimal-precision` witch is uses if the user has not specified a number precision
++ New parameter `hint` for verification event
```

Comment:
Please, add also a few words about the content for the hint. The check1 will use `hint` field to add information about the reason why the comparison was failed